src/Rings/localization_interface.jl

```diff
@@ -169,7 +179,7 @@ Domain:
 Multivariate polynomial ring in 3 variables over QQ
 Codomain:
 =========
-localization of Multivariate polynomial ring in 3 variables over QQ at the complement of ideal(x)
+Localization of multivariate polynomial ring in 3 variables over QQ at Complement of prime ideal
```

Comment:
Are all changes in the use of capital letters here intentional?

Reply:
Partly. But this map type comes from hecke and is not touched here. In the transition phase things will be inconsistent.

Reply:
It should just print the codomain using `print(io, codomain(...))`, so it should work?